### ISSUE TITLE
ProcessHub

### ISSUE THREAD

**Issue by tilor** [processhub/web#1]
Hello,
I'd like to contact you concerning your use of the ProcessHub name on GitHub.

**Comment by anas-sarairah** [comment]
Hi 
Yes , tell me what's up? 

Regards 
Anas

**Comment by tilor** [comment]
Hi Anas,
Thanks for getting back so quickly! We will release a new product on processhub.com soon and we also own the corresponding ProcessHub trademark. 
It seems you are not using your processhub repository and we would like to publish some open source components on GitHub - would you consider to transfer the repo?
Thanks & Greetings from Germany
Tilo

**Comment by tilor** [comment]
Is there any way to contact you personally to discuss?
Thanks!

**Comment by anas-sarairah** [comment]
Hi Tilor,

Am sorry I couldn't reply to you quickly but I was off work.

Any way I also have some open source business process management components that I have plans to release once I have a stable version and I have spent so much time choosing a name for the components bundle "processhub", so it's really hard for me to sell out this name, I also have a partner who refused to sell out the name.

I really wanna help you out specially that you already have a brand with that name but I will have to at least convince my partner that we can choose another name.
 
Tell me what do you think ? 

Best regards
Anas

**Comment by tilor** [comment]
Let's discuss on private mail, ok? info@processhub.com

**Comment by tilor** [comment]
BTW: Where are you from? Europe?

**Comment by anas-sarairah** [comment]
No am in Jordan ? Ok here is my skype anas.alsarairah

**Comment by tilor** [comment]
Hi Anas,

Sorry, I'm not on Skype. Jordan - cool!
The problem is you won't be able to use processhub for anything related to process management, at least if you use it internationally (which I guess you already do, GitHub servers should be in the US).
We own the legal ProcessHub trademark (check https://www.tmdn.org/tmview/welcome.html?lang=en and search for processhub) and are currently expanding that trademark internationally.
Anyhow I'd like to make this easy (easy things are usually good). How about I send you $200 via PayPal? That should be around half of what a lawyer would cost me in the first step.

Best, 

Tilo

**Comment by tilor** [comment]
Hi Anas,

I changed my last comment - it's not nice to immediately use the trademark hammer on someone, sorry.

Best,

Tilo

**Comment by anas-sarairah** [comment]
Hi Tilor,

It's ok things happen . Any way I will get back to you once I get a reply from my partner.

Thanks 
Anas

**Comment by anas-sarairah** [comment]
Dear Tilor
We are ready to let the name go for 500.00 (five hundred)USD, is that ok with you ? 

Regards
Anas

**Comment by tilor** [comment]
Hi Anas,

That's more than twice the amount I offered...  

Can I send payment with Paypal? 

Regards,

Tilo

**Comment by anas-sarairah** [comment]
Hi Tilor 
Yes I know but the name has been chosen by my partner so I can't do anything .

Yes you can send IT through the PayPal to my account anas.alsarairah@gmail.com .

But please can you tell me how to send the name to you using github , I haven't done this before.

Regards 
Anas

**Comment by tilor** [comment]
Hi Anas,
I have just sent the payment. I also haven't done this before: I guess the name is tied to the accont, so I guess you could send me the password for the login so I can take over?
We'll have to try...
If you want to send the password: info@processhub.com 
Thanks!
Tilo

**Comment by tilor** [comment]
https://help.github.com/articles/transferring-organization-ownership/

**Comment by anas-sarairah** [comment]
Yes I got the money , so let me now transfers the account using the link you sent me . Please stay online until I sent it to you

**Comment by anas-sarairah** [comment]
What's you name on GitHub

**Comment by anas-sarairah** [comment]
Are you there

**Comment by tilor** [comment]
Hi, I'm back. My GitHub name is "tilor"

**Comment by anas-sarairah** [comment]
Ok I have sent you the invitation , please make sure you recover it so that I make you owner and then I leave the org

**Comment by tilor** [comment]
I accepted, but I'm just listed as member, NOT as owner on https://github.com/orgs/processhub/people
Please make sure to make me an owner before you leave.

**Comment by anas-sarairah** [comment]
Yes now next step I will make you an owner

**Comment by anas-sarairah** [comment]
Ok now I moved you as owner . Please confirm

**Comment by tilor** [comment]
Ok, looks good. Thanks!
What time is it in Jordan?? Should be pretty late there!

**Comment by anas-sarairah** [comment]
Perfect , yeh it's late here about 2:00 AM but no problem I should serve you .
Any way I am coding right now , IT peeps has no sleep. 

Please remove me from the organization .

Regards
Anas

**Comment by tilor** [comment]
Ok, thanks, that worked out nicely!
Regards,
Tilo

**Comment by tilor** [comment]
I'll also remove this public thread now.

**Comment by anas-sarairah** [comment]
Great , thank you
